CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+Features in Alpha or Beta stage are tagged as such. We try to follow the same
+conventions as Kubernetes for [feature development
+stages](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions).
+
+## Unreleased
+
+## 0.1.0 - 2019-11-18
+
+### Added
+
+- Support for configuring and cleaning-up Pod networking as per the [CNI spec](https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md) [Beta].
```

Comment:
We should mention VXLAN, Geneve tunnels.

Reply:
Done